### PR TITLE
Linear Search in dart language

### LINKED ISSUE #2179: Linear Search
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [x] Dart - @keshakaneria
- [x] Go
- [x] Java
- [x] Julia
- [x] JS
- [x] Kotlin
- [x] Php
- [x] Python
- [x] Readme
- [x] Ruby
- [x] Rust - @raksha009
- [x] TypeScript - @mruna7

### PR DESCRIPTION
**Pull request by keshakaneria**
Linear search algorithm in dart

Fixes #2179 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Languages Used:
- Dart

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by keshakaneria** [comment]
@jainaman224 have a look at the code and suggest if corrections required.

File: Linear_Search/Linear_Search.dart [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	for(int i=0;i<list.length;i++){
+		if(x==list[i]){
+		 return "Found 8 at Index $i";
+		}
+	}
+	return "Not found!";
+}
+
+main(){
+	List<int> list = [0, 2, 5, 8, 10, 12, 15, 30];
```
Comment (raghav-dalmia) [inline Linear_Search/Linear_Search.dart]: take input dynamically

File: Linear_Search/Linear_Search.dart [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	print(linearSearch(list,8));
+}
+
+/*
```
Comment (raghav-dalmia) [inline Linear_Search/Linear_Search.dart]: add sample input

File: Linear_Search/Linear_Search.dart [hunk 3]
```diff
@@ -0,0 +1,18 @@
+String linearSearch(List<int> list, int x){
+	for(int i=0;i<list.length;i++){
```
Comment (raghav-dalmia) [inline Linear_Search/Linear_Search.dart]: add space around operators(;><=)

**Comment by raghav-dalmia** [comment]
Also, squash your commits

**Comment by keshakaneria** [comment]
Above changes will be done soon. Can you tell me how to squash commit?

**Comment by keshakaneria** [comment]
@raghav-dalmia Now code is written in dynamic (user input) values, kindly check if corrections.

**Comment by keshakaneria** [comment]
any corrections? @raghav-dalmia

**Comment by keshakaneria** [comment]
@singh-shreya6 Done, removed here too. Have a check :)

**Comment by singh-shreya6** [comment]
Please squash your commits @keshakaneria

**Comment by singh-shreya6** [comment]
Please squash your commits @keshakaneria

**Comment by singh-shreya6** [comment]
are you working on this? If not, I will close the PR.

**Comment by keshakaneria** [comment]
@singh-shreya6 yes i am will send a new PR as work is already done and i am unable to squash the commits so the new PR will have final work done on the code according to changes requested.

**Comment by keshakaneria** [comment]
I have squashed commits and also changed requested are already edited. So now you can merge and evaluate me.

**Comment by singh-shreya6** [comment]
The commit count is still 7 as you can see in the 2nd tab next to conversation here. Please squash your commits properly @keshakaneria

**Pull request merged**